### PR TITLE
Update design of Settings About page

### LINKED ISSUE #572: Dev Home Settings: About page craftsmanship & Related links Footer backplate update
### Dev Home version

_No response_

### Windows build number

_No response_

### Other software

_No response_

### Steps to reproduce the bug

Description
When the About page specs list item is collapsed, it should also hide the related links footer. Additionally, There's been a figma bug that's been updated so wanted to call it out here as well. The 'Related links' footer should be the secondary backplate color and it does appear to be a not as tall as spec'd. Attaching a image of the Windows 11 settings app for fit and finish reference

### Expected result

_No response_

### Actual result

_No response_

### Included System Information

_No response_

### Included Extensions Information

_No response_

### PR DESCRIPTION
**Pull request by krschau**
## Summary of the pull request
Go from this:
image

To this:
image

## PR checklist
- [ ] Closes #572
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

**Review by guimafelipe — APPROVED** [review]

File: settings/DevHome.Settings/Views/AboutPage.xaml [hunk 1]
```diff
@@ -25,18 +26,32 @@
                 <labs:SettingsExpander.HeaderIcon>
                     <BitmapIcon ShowAsMonochrome="False" UriSource="/Assets/DevHome.ico" />
                 </labs:SettingsExpander.HeaderIcon>
-                <TextBlock
-                    Foreground="{ThemeResource TextFillColorSecondaryBrush}"
-                    IsTextSelectionEnabled="True"
-                    Text="{x:Bind ViewModel.VersionDescription, Mode=OneWay}" />
+                <Button x:Uid="Settings_About_Copy" Padding="30,5,30,6" >
+                    <i:Interaction.Behaviors>
```
Comment (AmelBawa-msft) [inline settings/DevHome.Settings/Views/AboutPage.xaml:30]: A `Button` has a `Command` property

Reference: [Button Properties](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.button?view=windows-app-sdk-1.3#properties)

**Review by AmelBawa-msft — APPROVED** [review]

**Comment by niels9001** [comment]
@krschau We have worked with the design folks to create a consistent version card across all 1P apps - that is inline with the 'before' version. (Only difference is that the SettingsCard subtitle would be an all rights reserved message.

For reference, check out the Windows Visual Library Figma file (UI samples page) or see other 1P apps like Store, Photos, Media Player.

Is there a specific reason why we'd be moving away from that? It would cause additional UX inconsistencies across Windows..

**Comment by krschau** [comment]
@niels9001 Our designers opened bugs on what we have because it doesn't align with their Figmas.

**Comment by niels9001** [comment]
> @niels9001 Our designers opened bugs on what we have because it doesn't align with their Figmas.

Looking at this issue date, it looks like it was opened before the release and prior PR addressing this page? I know there was some discussion about it where we referenced the common UX pattern for this across all Microsoft apps.. might be that the Dev Home Figma wasn't updated since we fixed it on the spot?

**Comment by krschau** [comment]
@niels9001 I started an email thread about this, you'll see the most recent ask has come quite recently. (Sorry to move the discussion but I don't think the designer checks GithHub.) I won't check any of these changes in without making sure we're all on the same page.

**Comment by krschau** [comment]
Closing in favor of #1438

**Comment by krschau** [comment]
This becomes relevant again with #1502

**Pull request closed without merging**